usr/src/cmd/smbsrv/smbd/server.xml

```diff
@@ -229,6 +229,8 @@ file.
 			value='true' override='true'/>
 		<propval name='max_protocol' type='astring'
 			value='' override='true'/>
+		<propval name='encrypt' type='astring'
+			value='disabled' override='true'/>
```

Comment:
Is there a reason this is astring but a lot of other toggels are boolean? (Might be to be able to set it on/off/optional... but that is not immediately clear... I might have missed something though)

Reply:
nvm, just found the man page update I missed the first time going over this.